controllers/imagehub/controllers/data_helper.go

```diff
@@ -108,19 +110,25 @@ func (r *DataHelper) genFulldataByImageName(ctx context.Context, n imagehubv1.Im
 	fd := imagehubv1.FullData{}
 
 	orgInfo, err := r.getOrgInfoByOrgName(ctx, n.ToOrgName())
-	if err != nil {
+	if err == ErrNotMatch {
+		r.Logger.V(2).Info("getOrgInfoByOrgName", "err:", err.Error())
```

Comment:
failed to get origination info by name: %s, name

Reply:
done at [pr 2233](https://github.com/labring/sealos/pull/2233)